src/index.js

```diff
@@ -19,46 +24,65 @@ function isIPFS (addr) {
   }
 }
```

Comment:
https://github.com/multiformats/js-mafmt/issues/25 was resolved - this should now be unnecessary.

Reply:
Thanks